+ JOB=9
+ NO_JOBS=16
+ [[ ! -n 9 ]]
+ [[ ! -n 16 ]]
+ [[ 9 == global ]]
+ set +x
+ pkgcheck -r gentoo --reporter FancyReporter 'dev-util/*' 'x11-libs/*' 'kde-base/*' 'sci-mathematics/*' 'dev-tex/*' 'app-eselect/*' 'net-print/*' 'net-ftp/*' 'net-nntp/*' 'gnustep-libs/*' -d imlate -d unstable_only -d cleanup -d stale_unstable -d UnusedGlobalFlags -d UnusedLicense --profile-disable-dev --profile-disable-exp
WARNING pkgcore: invalid glsa- glsa-201206-24.xml, package www-servers/tomcat:
               : error range rlt version 5.5.34 is a guaranteed empty set

dev-util/aap
  DeprecatedEAPI: version 1.076: uses deprecated EAPI: 0

dev-util/ald
  DeprecatedEAPI: version 0.1.7: uses deprecated EAPI: 0

dev-util/android-ndk
  DeprecatedEAPI: version 8b: uses deprecated EAPI: 3
  DeprecatedEAPI: version 8c: uses deprecated EAPI: 3
  DeprecatedEAPI: version 8d: uses deprecated EAPI: 3
  DeprecatedEAPI: version 8e: uses deprecated EAPI: 3
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

dev-util/antlrworks
  DeprecatedEAPI: version 1.2.3: uses deprecated EAPI: 2
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

dev-util/appdata-tools
  DroppedKeywordWarning: version 0.1.8-r1: keywords dropped: alpha

dev-util/argouml
  DeprecatedEAPI: version 0.34: uses deprecated EAPI: 3

dev-util/aruba
  DroppedKeywordWarning: version 0.8.1: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-util/atomic-install
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

dev-util/autoproject
  DeprecatedEAPI: version 0.20: uses deprecated EAPI: 0
  DeprecatedEAPI: version 0.20_p5: uses deprecated EAPI: 0

dev-util/bcpp
  DeprecatedEAPI: version 20050725: uses deprecated EAPI: 0
  DeprecatedEAPI: version 20090630: uses deprecated EAPI: 2

dev-util/bdelta
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

dev-util/bitcoin-tx
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

dev-util/boost-build
  DeprecatedEclass: version 1.52.0-r1: uses deprecated eclass(es): [ python (migrate to python-r1 / python-single-r1 / python-any-r1) ]
  DeprecatedEclass: version 1.53.0: uses deprecated eclass(es): [ python (migrate to python-r1 / python-single-r1 / python-any-r1) ]
  DeprecatedEclass: version 1.54.0: uses deprecated eclass(es): [ python (migrate to python-r1 / python-single-r1 / python-any-r1) ]
  DeprecatedEclass: version 1.55.0: uses deprecated eclass(es): [ python (migrate to python-r1 / python-single-r1 / python-any-r1) ]

dev-util/btyacc
  DeprecatedEAPI: version 3.0-r2: uses deprecated EAPI: 2

dev-util/buildbot
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

dev-util/buildbot-slave
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

dev-util/bunny
  DeprecatedEAPI: version 0.93-r1: uses deprecated EAPI: 2

dev-util/bustle
  DroppedKeywordWarning: version 0.4.7-r1: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-util/catalyst
  DeprecatedEclass: version 2.0.9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]
  DeprecatedEclass: version 2.9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

dev-util/ccache
  DeprecatedEAPI: version 2.4-r9: uses deprecated EAPI: 0

dev-util/ccmalloc
  DeprecatedEAPI: version 0.4.0-r2: uses deprecated EAPI: 0

dev-util/cdecl
  DeprecatedEAPI: version 2.5-r2: uses deprecated EAPI: 0
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

dev-util/cgvg
  DeprecatedEAPI: version 1.6.2: uses deprecated EAPI: 0

dev-util/checkstyle
  DeprecatedEAPI: version 4.4: uses deprecated EAPI: 2
  DroppedKeywordWarning: version 5.4: keywords dropped: ppc

dev-util/cligh
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

dev-util/cmake
  DroppedKeywordWarning: version 3.2.2: keywords dropped: ia64, sparc
  DroppedKeywordWarning: version 3.2.3: keywords dropped: alpha, ppc

dev-util/cmocka
  DroppedKeywordWarning: version 0.4.1: keywords dropped: alpha

dev-util/coccigrep
  DeprecatedEclass: version 1.10: uses deprecated eclass(es): [ distutils (migrate to distutils-r1), python (migrate to python-r1 / python-single-r1 / python-any-r1) ]
  DeprecatedEclass: version 1.11: uses deprecated eclass(es): [ distutils (migrate to distutils-r1), python (migrate to python-r1 / python-single-r1 / python-any-r1) ]

dev-util/cocom
  DeprecatedEAPI: version 0.996-r1: uses deprecated EAPI: 2

dev-util/codeblocks
  MetadataError: version 12.11: attr(fetchables): error- Metadata Exception: pkg ebuild src: dev-util/codeblocks-12.11, attr fetchables
error: mirror://berlios/codeblocks/codeblocks_12.11-1.tar.gz is unparseable
flagged token- uri mirror://berlios/codeblocks/codeblocks_12.11-1.tar.gz isn't a known mirror tier
  MetadataError: version 13.12: attr(fetchables): error- Metadata Exception: pkg ebuild src: dev-util/codeblocks-13.12, attr fetchables
error: mirror://berlios/codeblocks/codeblocks_13.12-1.tar.gz is unparseable
flagged token- uri mirror://berlios/codeblocks/codeblocks_13.12-1.tar.gz isn't a known mirror tier

dev-util/colorgcc
  DeprecatedEAPI: version 1.3.2-r4: uses deprecated EAPI: 0
  DeprecatedEAPI: version 1.3.2-r5: uses deprecated EAPI: 3

dev-util/comparator
  DeprecatedEclass: version 2.5: uses deprecated eclass(es): [ distutils (migrate to distutils-r1), python (migrate to python-r1 / python-single-r1 / python-any-r1) ]
  DeprecatedEAPI: version 2.5: uses deprecated EAPI: 3
  DeprecatedEclass: version 2.9: uses deprecated eclass(es): [ distutils (migrate to distutils-r1), python (migrate to python-r1 / python-single-r1 / python-any-r1) ]

dev-util/confix-wrapper
  DeprecatedEAPI: version 0: uses deprecated EAPI: 0
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

dev-util/cproto
  DeprecatedEAPI: version 4.7e: uses deprecated EAPI: 0

dev-util/cpuinfo-collection
  DeprecatedEAPI: version 20060720: uses deprecated EAPI: 0
  DeprecatedEAPI: version 20070104: uses deprecated EAPI: 0

dev-util/crow-designer
  DeprecatedEAPI: version 2.21.0: uses deprecated EAPI: 3
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fdba9d28b10> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append